nyt-top-20-science-news/src/components/NewsFeed/NewsFeed.test.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import NewsFeed from './NewsFeed';
+
+let container = null;
+beforeEach(() => {
+  // setup a DOM element as a render target
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+it('renders with or without a valid news data', () => {
+  const NOT_FOUND_MESSAGE = 'No results found 👀 . Please refresh this page or try another search.';
+  const FAKE_NEWS = [{
+    title: 'fake title',
+    abstract: 'fake abstract',
+    byline: 'fake writer',
+    section: 'fake section',
+    updated_date: 'fake update date time',
+    multimedia: 'hteeteep://fake-news.com',
```

Comment:
Seeing your fake "hteeteep" protocol I thought you might enjoy this little tidbit: have you ever visited [example.com](http://example.com/)? If you go to it you'll see it's a very basic site but it's actually operated by [IANA](https://www.iana.org/about) and is reserved to be a reliably unreachable example url. And this has been a mandated rule for the core of the internet since the beginning of internet time. bizarre.

Reply:
Lol I have never bothered to actually go to example.com but I see it in most documentations. In hindsight, it makes sense to find out the reason behind that. Good to know. Thanks!